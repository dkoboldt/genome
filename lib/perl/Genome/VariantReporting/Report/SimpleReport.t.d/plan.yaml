reports:
    simple:
        interpreters:
            position: {}
            vep: {}
            "variant-type": {}
