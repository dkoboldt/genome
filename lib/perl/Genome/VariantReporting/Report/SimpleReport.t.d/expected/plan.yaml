---
experts: {}
needs_translation: 1
reports:
  simple:
    filters: {}
    interpreters:
      position: {}
      variant-type: {}
      vep: {}
